apps/protocol-frontend/src/components/AttestationsTable.tsx

```diff
@@ -39,6 +24,7 @@ import { displayAddress } from '../utils/web3';
 import { RowSelectionState } from '@tanstack/table-core';
 import { statusEmojiSelect } from '../utils/statusEmojiSelect';
 import { formatDate, toDate } from '../utils/date';
+import GovrnTable from './GovrnTable';
```

Comment:
(Arbitray comment)

I am testing out the sorting and it seems to break on the contribution table on engagement date. When I go for pagination it unselects my sorting preference.

[Screencast from 2023-02-02 12-05-57.webm](https://user-images.githubusercontent.com/12705146/216393126-2d50b5f3-2a19-4da3-89fc-94902b844e40.webm)